{{cookiecutter.project_shortname}}/Dockerfile

```diff
@@ -2,8 +2,10 @@
 FROM {{cookiecutter.project_shortname}}-base:latest
 
 COPY ./ .
-COPY ./docker/uwsgi/ ${INVENIO_INSTANCE_PATH}
 
-RUN ./scripts/bootstrap
+RUN pipenv run pip install .
 
-USER ${INVENIO_USER_ID}
+RUN pipenv run invenio collect -v
```

Comment:
Should this not be using the bootstrap script?

Reply:
We could parametrize the `bootstrap` script to address different use cases (like we do e.g. with `--ci`) in order to keep this logic in one place

Reply:
This is doing the same but was changed from running the `bootstrap` script because of the `--unsafe` flag that is passed in `webpack install. The flag is needed because of the root user, could also be changed to `--user=root` with the same effect